includes/controls/gaps.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor gap control.
+ *
+ * A base control for creating gap control. Displays input fields for two values,
+ * row/column, hight/width/ and the option to link them together.
```

Comment:
Suggested: 'A base control for creating a gap control. Displays input fields for two values,
 row/column, height/width and the option to link them together'

Reply:
@hein-obox Done